frontend/src/app/shared/form/oib-scan-mode/oib-scan-mode.component.html

```diff
@@ -1,6 +1,12 @@
-<div class="form-group" *ngIf="settings">
+<div class="form-group" *ngIf="settings && !settings.subscriptionOnly">
   <label [for]="'south-specific-' + settings.key">{{ settings.label }}</label>
   <select [formControlName]="settings.key" [id]="'south-specific-' + settings.key" class="form-select mb-2 mr-sm-2">
+    <option
+      *ngIf="settings.acceptSubscription || settings.subscriptionOnly"
+      [ngValue]="{id: 'subscribe', name: '', description: '', cron: ''}"
```

Comment:
you should have a field in the component to avoid reacreating the object every time that Angular checks the template